### DISCUSSION TITLE
Ability to sleep a tab/group of tabs

### DISCUSSION THREAD

**Discussion by flayks** [zen-browser/desktop#1406] [Ideas]
I'm currently using the Stack Next browser and they have the feature to sleep tabs that we might not need. In my case, I have a few Slack instances open in tabs and it is very convenient to only have the ones I need active. Also, the Arc browser does a similar thing with favorites, when they're not open they'd just not be active but stay in the sidebar.

I tried some Firefox extensions to replicate that but it's not the same as it would sleep all the tabs and not one in particular.

For instance here, only the first Slack instance is active, the rest is sleeping.

They use the shortcut Cmd+S which I find very convenient!

![Screenshot 2024-09-12 at 20 18 34@2x](https://github.com/user-attachments/assets/398446dc-7f85-485c-b3dc-bc47ebf40fd9)

**Comment by Muko-Tabi** [comment]
You could try using this [extension](https://addons.mozilla.org/en-US/firefox/addon/auto-tab-discard/) and as for why the sleeping function in Firefox isn't the same from your is mostly Firefox engine sleeping function is different from Chromium engine

**Reply by flayks** [reply]
I think I saw this extension but indeed you cannot sleep only one tab, just all of them which I'm not a fan of

**Reply by Muko-Tabi** [reply]
![image](https://github.com/user-attachments/assets/b1a07226-9a0a-4c6e-ae3e-25c1de81892e)
You can right click single tab or multiple tab to discard (make them sleep), Discard tab is the current tab and Discard tab group is multiple tabs of your choice

**Reply by flayks** [reply]
Indeed that works as well! It's just a bit deeper to find out, a shortcut or more integrated way would be nicer imo!
